internal/acceptance/external_location_test.go

```diff
@@ -60,3 +60,112 @@ func TestUcAccExternalLocationForceDestroy(t *testing.T) {
 		}`,
 	})
 }
+
+func TestUcAccExternalLocationUpdate(t *testing.T) {
+	unityWorkspaceLevel(t, step{
+		Template: `
+		resource "databricks_storage_credential" "external" {
```

Comment:
It's very hard to see the differences between these steps. Could we refactor this a bit so that the differences are more obvious? E.g. a function (or a few functions) to construct the different resources involved in these templates.

Reply:
updated